GTF_Xp/Managers/CustomScalingBuffManager.cs

```diff
@@ -10,7 +10,7 @@ public static class CustomScalingBuffManager
     {
         public static void ApplyCustomScalingEffects(PlayerAgent targetAgent, List<CustomScalingBuff> buffs)
         {
-            if (buffs is null)
+            if (buffs is null || !targetAgent.IsLocallyOwned)
```

Comment:
Hey, all looks good. But i think this IsLocallyOwned check should atleast also be done in the beginning of the SetCustomBuff method.
You may still get into the SetCustomBuff method with a non local PlayerAgent by calling the public method ResetCustomBuffs as an example.